src/deltacode/__init__.py

```diff
@@ -25,62 +25,65 @@
 
 from __future__ import absolute_import
 
+import os
 from collections import OrderedDict
 
-from deltacode.models import File
-from deltacode.models import Scan
 from deltacode import utils
+from deltacode.exceptions import FileError as FileError
```

Comment:
```suggestion
from deltacode.exceptions import FileError
```
I don't think we need to import FileError as FileError?